m_proxy/flask/main.py

```diff
@@ -46,6 +47,24 @@ def proxy_post():
     """ Proxy to DialogFlow
     """
     secrets = get_secrets()
-#    req_data = request.get_json()
+    req_data = request.get_json()
 
-    return jsonify("")
+    project = req_data['project'] #project id
+    user_query = req_data['user_query'] #user question
+
+    session_id = '123456789' #randomized for every session
```

Comment:
Do we have to explicitely randomize this? 

It looks like "using the same session id allows continuation of the conversation". Maybe this has to be generated using something like `uuid.uuid1()` the session_id value returned as part of the response and also accepted as part of the query input.

If someone passes a session_id in the input it will continue their conversation.

Does this sound right?

Reply:
Yeah we don't have to explicitly randomize it, I just did that to make things clear and was also searching for a good simple randomization method but it looks like you already know that one for us to use.